addons/website_sale/views/templates.xml

```diff
@@ -1841,7 +1841,7 @@
                         <t t-out="payment_tx_id.acquirer_id.sudo().cancel_msg"/>
                     </t>
                     <t t-if="payment_tx_id.state == 'authorized'">
-                        <span>Your payment has been authorized.</span>
+                        <span t-if='payment_tx_id.acquirer_id.sudo().auth_msg' t-out="payment_tx_id.acquirer_id.sudo().auth_msg"/>
```

Comment:
```suggestion
                        <span t-if="payment_tx_id.acquirer_id.sudo().auth_msg" t-out="payment_tx_id.acquirer_id.sudo().auth_msg"/>
                        <span t-else="">Your payment has been authorized.</span>
```

Just to avoid disrupting the existing implementations.